tensorflow/lite/micro/tools/make/check_optimized_kernel_dir.sh

```diff
@@ -31,7 +31,17 @@
 set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-ROOT_DIR=${SCRIPT_DIR}/../../../../..
+
```

Comment:
let's document the new parameter at the top of the file.

```
# Called with following arguments:
# 1 - OPTIMIZED_KERNEL_PATH (relative to TF root) to the optimized kernel implementations.
# 2 - (optional) TENSORFLOW_ROOT: path to root of the TFLM tree (relative to directory from where the script is called.
```

Reply:
I'm a bit confused here. OPTIMIZED_KERNEL

```make
OPTIMIZED_KERNEL_DIR_PREFIX := $(TENSORFLOW_ROOT)tensorflow/lite/micro/kernels
PATH_TO_OPTIMIZED_KERNELS := $(OPTIMIZED_KERNEL_DIR_PREFIX)/$(OPTIMIZED_KERNEL_DIR)
RESULT := $(shell $(MAKEFILE_DIR)/check_optimized_kernel_dir.sh $(PATH_TO_OPTIMIZED_KERNELS) $(TENSORFLOW_ROOT))
```

what this means is that the parameter to check_optimized_kernel_dir.sh includes TENSORFLOW_ROOT. So we should not need to `cd ${2}`.

I think we should modify the script to take the OPTIMIZED_KERNEL_PATH parameter be relative to the directory from where the script is called (instead of relative to TF_ROOT). Then we won't need any if in the script.

Reply:
done.

Reply:
Changed according to our discussion.